start.sh

```diff
@@ -120,7 +119,8 @@ EOI
     cat << EOI
 
 ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ Do you want to change the IPFS settings [y/n]:                                              ┃
+┃ Do you want to change the IPFS settings [y/N]:                                              ┃
+┃ [When using Testnet(Paseo), you must change the IPFS Provider information]                  ┃
```

Comment:
Is this true? Perhaps we need to adjust it at least? The [tutorial](https://github.com/ProjectLibertyLabs/gateway/pull/346) would need to be longer to support this statement as currently it assumes you don't have to do this.

Perhaps something like this?
```suggestion
┃ [For persistence of data, you must change the IPFS Provider information]                  ┃
```

Or perhaps 

```suggestion
┃ [Suggestion: Change to an IPFS Pinning Service for better persistence and availability]                  ┃
```